test/test_dbgfile.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import os
+
+from elftools.elf.elffile import ELFFile, DynamicSection
+from elftools.dwarf.callframe import ZERO
+
+class TestDBGFile(unittest.TestCase):
+    def test_dynamic_segment(self):
+        with open(os.path.join('test', 'testfiles_for_unittests',
+                               'debug_info.elf'), 'rb') as f:
+            elf = ELFFile(f)
+            
+            for segment in elf.iter_segments():
+                if segment.header.p_type != 'PT_DYNAMIC':
+                    continue
+
+                assert segment.num_tags() == 0, "The dynamic segment in this file should be empty"
+                break
+            else:
+                assert False, "There should be a dynamic segment in this file"
```

Comment:
Here and elsewhere in the file is there an advantage to using `assert` and not standard test assertion methods?

Reply:
Ah, right. I'm used to using `nose`, and my default upon not seeing it in the other tests was to fall back to the default assertions